terraform/layer2-k8s/eks-cluster-autoscaler.tf

```diff
@@ -30,5 +30,5 @@ resource "helm_release" "cluster_autoscaler" {
     data.template_file.cluster_autoscaler.rendered,
   ]
 
-  depends_on = [helm_release.prometheus_operator]
+  depends_on = [module.kube_prometheus_stack]
```

Comment:
what will be if we set `kube_prometheus_stack_enable = false` ?

Reply:
In this case `kube-prometheus-stack and dependent` charts will not be installed.  Right now it's a hard dependency. Next we will prepare modules for `cluster-autoscaler`, `nginx-ingress` and `loki-stack`, in modules we will add possibility to enable `service-monitor` for these charts depending on `kube_prometheus_stack_enable`.